packages/plugins/test/e2e/SafeSponsorEverythingPaymaster.test.ts

```diff
@@ -0,0 +1,112 @@
+import { expect } from "chai";
+import { ethers } from "ethers";
+import {
+  SafeECDSAFactory__factory,
+  SafeECDSAPlugin__factory,
+  SponsorEverythingPaymaster__factory,
+  EntryPoint__factory
+} from "../../typechain-types";
+import receiptOf from "./utils/receiptOf";
+import { setupTests } from "./utils/setupTests";
+import { createAndSendUserOpWithEcdsaSig } from "./utils/createUserOp";
+
+const oneEther = ethers.parseEther("1");
+
+describe("SafePaymasterPlugin", () => {
+  it("should pass the ERC4337 validation", async () => {
+    const {
+      bundlerProvider,
+      provider,
+      admin,
+      owner,
+      entryPointAddress,
+      deployer,
+      safeSingleton,
+    } = await setupTests();
+
+    // Deploy paymaster.
+    const paymaster = await deployer.connectOrDeploy(
+      SponsorEverythingPaymaster__factory,
+      [entryPointAddress],
+    );
+    const paymasterAddress = await paymaster.getAddress();
+    
+    // Paymaster deposits.
+    await paymaster.deposit({ value: ethers.parseEther("1") })
```

Comment:
Minor suggestion
```suggestion
    await paymaster.deposit({ value: oneEther })
```
Can also do this in a few other spots.